File: solutions/c/easy/too-high-boxes/solution-approach1.c
// ──────────────────────────────────────────────────
// Link        https://www.hackerrank.com/challenges/too-high-boxes/problem?isFullScreen=true
// Problem     Boxes through a Tunnel
// Difficulty  Easy
// Subdomain   Structs and Enums
// Platform    HackerRank
// Language    c
// Status      Accepted
// Submitted   2026-07-17, 09:56 p.m.
// ──────────────────────────────────────────────────



struct box
{
	/**
	* Define three fields of type int: length, width and height
	*/
    int length;
    int width;
    int height;

};

typedef struct box box;

int get_volume(box b) {
	/**
	* Return the volume of the box
	*/
    return b.length * b.width * b.height;
}

int is_lower_than_max_height(box b) {
	/**
	* Return 1 if the box's height is lower than MAX_HEIGHT and 0 otherwise
	*/
    if (b.height < MAX_HEIGHT) {
        return 1;
    } else {
        return 0;
    }
}

File: solutions/python3/easy/swap-case/solution-approach1.py
# ──────────────────────────────────────────────────
# Link        https://www.hackerrank.com/challenges/swap-case/problem?isFullScreen=true
# Problem     sWAP cASE
# Difficulty  Easy
# Subdomain   Strings
# Platform    HackerRank
# Language    python3
# Status      Accepted
# Submitted   2026-08-10, 10:34 p.m.
# ──────────────────────────────────────────────────

def swap_case(s):
    swapped = []
    for char in s:
        if char.islower():
            swapped.append(char.upper())
        elif char.isupper():
            swapped.append(char.lower())
        else:
            swapped.append(char)
    return "".join(swapped)


